### DISCUSSION TITLE
"Hire" function in Mastodon

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#17097] [General]
I could imagine that if Mastodon had a "hiring" function, like Unsplash has implemented for photography, it could become more exciting, especially for freelancers. What do people from the Mastodon community think?

**Comment by linux-lukas** [comment]
Would ActivityPub be able to implement a "Hire" function for all Fediverse services?

**Comment by koyuawsmbrtn** [comment]
I'd say a DM should be enough, people on Facebook also get hired through Messenger

**Reply by linux-lukas** [reply]
This would certainly also be application-neutral - it would then only have to be the "hire" workflow based on the DMs (incl. corresponding labelling of the DM in web and mobile view)?
